lib/api/globals/index.ts

```diff
@@ -9,22 +9,81 @@ import {
   contactFormFragment,
 } from "@/lib/api/fragments/global";
 import { categoriesFragment } from "@/lib/api/fragments/categories";
-import { userProfileFragment } from "@/lib/api/fragments/page";
 import { getSiteFromLocale } from "@/lib/helpers/site";
 import tags from "../client/tags";
+import { getImageProps } from "next/image";
+import { cantoToImageProps } from "../canto";
 
-export async function getGlobalData(locale = fallbackLng) {
+export async function getLogos() {
```

Comment:
extract these queries from the global fetch and context, these no longer need to be using client memory in the context since they are server rendered.